app/decorators/decidim/initiatives/initiatives_controller_decorator.rb

```diff
@@ -11,7 +11,7 @@
   # Assigns the value "closed" to the :state key until the given date is reached.
   def default_filter_params
     original_default_filter_params.tap do |default_params|
-      default_params[:state] = "closed" if Time.current < SIGNATURE_START_DATE
+      default_params[:state] = "closed"
```

Comment:
The requested feature is to show "filed" initiatives after the `SIGNATURE_START_DATE` is reached.
Thus this should not be the default state.

Reply:
Done!